gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py

```diff
@@ -165,6 +165,8 @@ def _replace_macros(self, query):
     for macro in self._QueryMacros:
       if macro.name == query:
         return macro.value
+      if query.startswith(macro.name):
+        return macro.value + query[len(macro.name):]
```

Comment:
oh, this is very hacky :) it assumes the query ends with `{TABLE_NAME}` and the logic is cryptic.
Please just inline the entire query for your tests (it's ok to have duplicate logic in this case as it's a very specific use case).
`SELECT COUNT(0) AS num_rows FROM {TABLE_NAME}_chr01`, etc.